static/app/views/issueList/searchBar.tsx

```diff
@@ -121,15 +123,29 @@ function IssueListSearchBar({organization, tags, onClose, ...props}: Props) {
           : undefined,
         statsPeriod: pageFilters.datetime.period,
       };
+      const eventsDatasetValues = fetchTagValues({
+        api,
+        orgSlug,
+        tagKey: key,
+        search,
+        projectIds,
+        endpointParams,
+        dataset: Dataset.ERRORS,
+      });
 
-      return fetchTagValues({
+      const issuePlatformDatasetValues = fetchTagValues({
         api,
         orgSlug,
         tagKey: key,
         search,
         projectIds,
         endpointParams,
+        dataset: Dataset.ISSUE_PLATFORM,
       });
+
+      return await Promise.all([eventsDatasetValues, issuePlatformDatasetValues]).then(
```

Comment:
Look like this is mixing async/await and the old `.then()` syntax. Not the worst thing, but might be a bit cleaner to do 

```
const [eventTagValues, issuePlatformTagValues] = await Promise.all([...]);

return mergeAndSortTagValues(eventTagValues, issuePlatformTagValues);
```